examples/simple/README.md

```diff
@@ -4,8 +4,12 @@
 
 ### Install
 
+Start fetchr root:
+
 ```js
 npm install
+cd examples/simple
+npm install
```

Comment:
there are 2 npm installs here, I think thats a typo. also, the instructions are assuming the user is in the examples/simple folder already (since the readme is there). I'm not so sure its needed.